### PR TITLE
change version of apiregistration.k8s.io to v1

### LINKED ISSUE #274: The field `priority` does not exist in `apiregistration.k8s.io/v1beta`
when use kubectl apply -f config/apiserver.yaml, it throws following error:
```
error: error validating "config/apiserver.yaml": error validating data: ValidationError(APIService.spec): unknown field "priority" in io.k8s.kube-aggregator.pkg.apis.apiregistration.v1beta1.APIServiceSpec; if you choose to ignore these errors, turn validation off with --validate=false
```
Can i open a pr to remove this field?

### PR DESCRIPTION
**Pull request by magicsong**
fix #274

### CODE REVIEW

File: cmd/apiserver-boot/boot/build/build_resource_config.go [hunk 1]
```diff
@@ -272,7 +272,7 @@ type resourceConfigTemplateArgs struct {
 var resourceConfigTemplate = `
 {{ $config := . -}}
 {{ range $api := .Versions -}}
-apiVersion: apiregistration.k8s.io/v1beta1
```
Comment (yue9944882) [inline cmd/apiserver-boot/boot/build/build_resource_config.go]: thx for fixing this, update related yaml files under `example/` please

File: cmd/apiserver-boot/boot/build/build_resource_config.go [hunk 2]
```diff
@@ -283,7 +283,6 @@ spec:
   version: {{ $api.Version }}
   group: {{ $api.Group }}.{{ $config.Domain }}
   groupPriorityMinimum: 2000
-  priority: 200
```
Comment (yue9944882) [inline cmd/apiserver-boot/boot/build/build_resource_config.go]: set the version priority to 1 please since that 0 priority will fail validation

Comment (yue9944882) [reply cmd/apiserver-boot/boot/build/build_resource_config.go]: note that version priority should range in (0, 1000)

Comment (magicsong) [reply cmd/apiserver-boot/boot/build/build_resource_config.go]: it is 10 now. you should check the whole file

File: cmd/apiserver-boot/boot/build/build_resource_config.go (hunk 1 above)
Comment (magicsong) [reply cmd/apiserver-boot/boot/build/build_resource_config.go]: sure

**Pull request merged**